spec/appliance_console/database_replication_spec.rb

```diff
@@ -115,4 +115,33 @@
       expect(subject.generate_cluster_name).to be_falsey
     end
   end
+
+  context "#write_pgpass_file" do
+    let(:pgpass_path) { Tempfile.new("pgpass").path }
+
+    before do
+      stub_const("#{described_class}::PGPASS_FILE", pgpass_path)
```

Comment:
Ah I misread your comment before. I only changed the method, not the spec. Let me fix this up ...